neo/rawio/neuralynxrawio.py

```diff
@@ -98,19 +108,27 @@ def _parse_header(self):
             chan_names = info['channel_names']
             chan_ids = info['channel_ids']
 
-            for idx, chan_id in enumerate(chan_ids):
-                chan_name = chan_names[idx]
+            for idx, (chan_id, chan_name) in enumerate(zip(chan_ids, chan_names)):
+                # ^^^ I had never seen cases when idx is non-zero // Mike
+                
+                uid = gen_uid(fid, idx, chan_id)
+                self.uid2chid[uid] = (fid, idx, chan_id)
+
                 if ext == 'ncs':
-                    # a signal channels
+                    # signal channels
                     units = 'uV'
                     gain = info['bit_to_microVolt'][idx]
                     if info['input_inverted']:
                         gain *= -1
                     offset = 0.
                     group_id = 0
-                    sig_channels.append((chan_name, chan_id, info['sampling_rate'],
+
+                    ### Note: vvvv: this structure could contain non-unique chan_name, chan_id
+                    sig_channels.append((chan_name, uid, info['sampling_rate'],
```

Comment:
Let s go back to
sig_channels.append((chan_name, chan_id, info['sampling_rate'],
so the use can have something that match the real id in the file and the software.